client/rpcserver/handlers.go

```diff
@@ -424,6 +426,77 @@ func handleLogout(s *RPCServer, _ *RawParams) *msgjson.ResponsePayload {
 	return createResponse(logoutRoute, &res, nil)
 }
 
+// sortOrderBook converts a *core.OrderBook into an *Orderbook, setting
+// appropriate fields and sorting buys and sells by descending and ascending
+// rates respectively.
```

Comment:
A `core.OrderBook` as returned from `Book` is already sorted, and the only new field you're adding, `Immediate`, is always false for booked orders. I'm fairly certain you can drop this conversion and the `rpcserver.OrderBook` type altogether.